docs/rest-api/api/driveitem_get_content_format.md

```diff
@@ -85,3 +85,11 @@ See [Error Responses][error-response] for more info about how errors are returne
   "section": "documentation",
   "tocPath": "Items/Download formats"
 } -->
+
+### Known constraints with HTML to PDF conversion
+
+(1)	The component used to transform HTML to PDF is not as smart as modern web browsers at correctly parsing HTML that is not well-formed. For example, if a table row is missing an end tag (</tr>) this component is not smart enough to detect that the next open tag (<tr>) is the beginning of a new row. The component will render a table with one long row, much of which ends up being clipped beyond the page. For best performance validate that the HTML is well formed – ideally following the XHTML standard. 
```

Comment:
Since this would be public docs, please try to choose words that fit the tone of documentation -just mention the constraints as they are, and specify that HTML to PDF conversion works best if the HTML follows a particular standard.
If you haven't done so please add items to your backlog to address these known issues.

Reply:
Thanks. I submitted some updates to modify this verbiage accordingly, as well as to address the issues versus constraints piece above.